### ISSUE TITLE
Add Search to menu

### ISSUE THREAD

**Issue by rgilman** [ContextInstitute/bfcom#41]
At some point we will need to get search functioning on the site but for now we just need a placeholder in the top menu.

Since the top menu uses WordPress's Appearance > Menus functionality, each one of us needs to do this manually (for now). Here's what I've done.

1. Go to https://dev.bfcom.local/wp-admin/nav-menus.php or Dashboard > Appearance > Menus – however you like to think about it.
2. Under Pages, choose View All and find Site-Wide Activity and add that to the end of bfcom-main-menu.
3. Once added, open that menu pane and change the name to Search.
4. Save it.

You could point this menu to any page you like – for now it's just a placeholder.

Once added, we can include it in our design work for both desktop and mobile.

**Comment by iangilman** [comment]
@DavidScottBernstein @wunluv please follow the instructions above and then post here to let everyone know that you've done so. :-)

**Comment by rgilman** [comment]
Apologies for not being as clear as I might have been.

Here's a screenshot:

![screen shot 2018-10-01 at 1 27 44 pm](https://user-images.githubusercontent.com/615052/46313904-f6fa9000-c57d-11e8-8d2e-19977991588d.png)

In it, I have checked the checkbox for Site-Wide Activity. The next step is to click the "Add to Menu" button. 

Once added, the lower part of the menu list will look like this:

![screen shot 2018-10-01 at 1 32 32 pm](https://user-images.githubusercontent.com/615052/46314179-b0596580-c57e-11e8-947e-ee008d462d34.png)

Click the little disclose triangle next to the word "Page" on the right. The panel will open to look like this:

![screen shot 2018-10-01 at 1 32 48 pm](https://user-images.githubusercontent.com/615052/46314228-d54dd880-c57e-11e8-9c21-83fbad2521d1.png)

Change the "Navigation Label" from Site-Wide Activity to Search. The end result should look like this:

![screen shot 2018-10-01 at 1 33 01 pm](https://user-images.githubusercontent.com/615052/46314294-f9111e80-c57e-11e8-98c3-e3ca255262ef.png)

Once you've done all of that, remember to "Save Menu" (blue buttons on the right of the screen).

Hope that helps.

**Comment by DavidScottBernstein** [comment]
Yes, big help. Thank you. Done.

**Comment by DavidScottBernstein** [comment]
Yes. However, I don’t see a search menu item after I made the above change in Wordpress.

**Comment by rgilman** [comment]
@DavidScottBernstein do you mean that you don't see it on bfn.context.org or that you don't see it on your local dev site? As for bfn.context.org, that was my oversight. I've now added the Search menu.

**Comment by DavidScottBernstein** [comment]
@rgilman I didn't see Search on either. Now I see it in the top menu on bfn.context.org but still not on my local site after I fetched from Master.

**Comment by rgilman** [comment]
Hmm... Curious. The inclusion of Search in the menu should be controlled by the database and not the theme files so I'm puzzle about why it's disappeared from you local site. In any case, I suggest you go back to Admin > Dashboard > Appearance > Menus to see what's going on and redo the above procedure if need be.

**Comment by iangilman** [comment]
@DavidScottBernstein Sounds like maybe you never fully got the search menu on in the first place? Perhaps you missed one of the steps in the instructions above? Anyway, yes, trying again sounds like a good idea. Keep in mind that it's controlled entirely by what you do in the admin and should have nothing to do with the code.
